content/en/reference/deployment/config-file/targets.md

```diff
@@ -158,9 +179,72 @@ targets:
     namespace: overflow
     strategy: canary-wait-til-approved
     constraints:
-      dependsOn: ["dev-west"]
+      dependsOn: ["staging"]
       beforeDeployment:
         - pause:
-            duration: 60
-            unit: seconds
+            untilApproved: true
+            requiresRole:
+              - Organization Admin
+            approvalExpiration:
+              duration: 24
+              unit: hours
+        - runWebhook:
+            name: Send_Slack_Deployment_Approval_Required
 ```
+
+#### `targets.<targetName>.constraints.afterDeployment`
+
+Add conditions that must be met before deployment to this target is considered finished. 
+
+`afterDeployment` supports the same `pause` and `runWebhook` conditions as [`beforeDeployment`](#targetstargetnameconstraintsbeforedeployment). 
+
+In this example, there are four targets: `dev`,  `infosec`, `staging`, and `prod-west`. After you deploy code to `infosec` and `staging`, you want to run jobs against those targets. If either of those jobs fails, CD-as-a-Service does not deploy to `prod-west`.
+
```

Comment:
I believe we should add - If an afterDeployment constraint fails, that target is not rolled back, but all the subsequent environments are not deployed.

Reply:
Rewrote the section and added this